service/routes.py

```diff
@@ -165,8 +190,7 @@ def create_shopcart_items():
     shopcart_item.deserialize(request.get_json())
     shopcart_item.create()
     message = shopcart_item.serialize()
-    #TODO: location_url = url_for("get_shopcart_items", 
-    #                              shopcart_item_id=shopcart_item.id, _external=True)
```

Comment:
Can you leave this in? this code needs to be implemented as part of issue #36

Reply:
Added!

Reply:
Thanks